### ISSUE TITLE
Fielf Of View computation

### ISSUE THREAD

**Issue by devasur** [borismus/webvr-boilerplate#150]
https://w3c.github.io/webvr/#vrfieldofview

Do you have any plans to include a standardized device selection options (in lines of dbdb) to wbvr-manager?  Will be very useful for setting the FoVs of the scene correctly.

If you do not have any plans, how would you recommend going about incorporating this feature?

**Comment by borismus** [comment]
FOV depends mostly on the viewer parameters, not so much the phone. Take a
look at device-info.js in polyfill

**Comment by devasur** [comment]
Yes.  You are right.  I am still trying get that exact responsibility split arounf polyfill vs. manager constructs.  IMO, wouldn't the the VR app need to query for the Viewer FoV and then use 2xFoV as the the ThreeJS camera FoV for appropriate rendering?  This was my question.  So, if we were to do that the viewer selection need to expand beyond Carboard V1 & Cardboard V2, with an easy to extend and expand structure.  This is where a global database similar to dbdb will help.  Probably it can carry identifiers, a standard icon and the rest of the parameters (that profile generator creates?)  I am sure rest of the parameters are important for the polyfill to draw bevels correctly too.
